### PR TITLE
Lazily initialize `request.accept` 

### PR DESCRIPTION
**Pull request by initial-wu**
To save memory consumption.

### CODE REVIEW

File: lib/request.js [hunk 1]
```diff
@@ -462,6 +463,17 @@ module.exports = {
...
   },
 
+  /**
+   * Get accept object.
+   * Lazily memoized.
+   *
+   * @return {Object}
+   * @api public
+   */
+  get accept() {
```
Comment (dead-horse) [inline lib/request.js]: maybe you should add `set accept()` to avoid breaking change.

Comment (initial-wu) [reply lib/request.js]: All right.

Comment (initial-wu) [reply lib/request.js]: I've pushed another commit to this PR.

File: lib/request.js [hunk 2]
```diff
@@ -462,6 +463,27 @@ module.exports = {
...
+  },
+
+  /**
+   * Set accept object.
+   *
+   * @param {Object}
+   * @api public
+   */
+  set accept(obj) {
+    return this._accept = obj;
```
Comment (dead-horse) [inline lib/request.js]: need a test case

File: lib/context.js [hunk 3]
```diff
@@ -194,6 +194,7 @@ delegate(proto, 'request')
   .access('query')
   .access('path')
   .access('url')
+  .access('accpet')
```
Comment (fl0w) [inline lib/context.js]: Misspelled

Comment (initial-wu) [reply lib/context.js]: thx for reminding, i should have run a test

**Comment by initial-wu** [comment]
Added some test cases and reorganized the commits. @fl0w @dead-horse

**Pull request merged**